### PR TITLE
Many-to-many user relationship -> user column migration

### PR DESCRIPTION
**Pull request by samwho**
## Description

Implements the backend code required to migrate many-to-many user relationships.

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.11.38",
+  "version": "2.11.39",
```
Comment (mike12345567) [inline lerna.json:2]: not sure why this changed - you might need to pull master into the base branch.

File: packages/server/src/tests/utilities/api/index.ts [hunk 2]
```diff
@@ -27,5 +27,6 @@ export default class API {
     this.datasource = new DatasourceAPI(config)
     this.screen = new ScreenAPI(config)
     this.application = new ApplicationAPI(config)
+    this.global = new GlobalAPI(config)
```
Comment (mike12345567) [inline packages/server/src/tests/utilities/api/index.ts:30]: Just to check - is the `GlobalAPI` in the base branch?

**Review by mike12345567 — APPROVED** [review]
Looks good to me - very clear! Just a couple of type errors to check !

File: packages/server/src/tests/utilities/TestConfiguration.ts [hunk 3]
```diff
@@ -315,10 +315,7 @@ class TestConfiguration {
       roles,
     })
     await cache.user.invalidateUser(globalId)
-    return {
-      ...resp,
-      globalId,
-    }
+    return resp
```
Comment (mike12345567) [inline packages/server/src/tests/utilities/TestConfiguration.ts:318]: Why doesn't think include the `globalId` anymore?

Comment (samwho) [reply packages/server/src/tests/utilities/TestConfiguration.ts:318]: It wasn't used by anything (tests seem to pass without it there), and it was preventing the `User` type from being the correct return type of this function.

Comment (mike12345567) [reply packages/server/src/tests/utilities/TestConfiguration.ts:318]: Ah perfect in that case!

**Review by mike12345567 — APPROVED** [review]
LGTM!

**Pull request merged**